components/lists/FeedList.tsx

```diff
@@ -8,8 +16,90 @@ import { View } from 'react-native';
  * - add tests for the feed list component
  */
 
+interface FeedListItemProps {
+  item: Feed;
+}
+
+const { height } = Dimensions.get('screen');
+
+const FeedListItem = ({ item }: FeedListItemProps) => {
+  const { colors } = useTheme();
+  return (
+    <View style={styles.feed} testID="feedItem">
+      <StyledText size="md">{item.body}</StyledText>
+      <View style={styles.imageContainer}>
+        <Image style={styles.image} source={{ uri: item.image }} />
+      </View>
+
+      <Row>
+        <Link
+          href={{
+            pathname: '/home/feed/share',
+            params: { feed: item },
+          }}
+        >
+          <View style={styles.share}>
+            <StyledText size="md" allowFontScaling font="bold" variant="link" style={styles.shareLabel}>
+              Share
+            </StyledText>
+            <FontAwesome size={20} name="share" color={colors.primary} />
+          </View>
+        </Link>
+
+        <StyledText size="sm">5 hours ago</StyledText>
+      </Row>
+    </View>
+  );
+};
+
 const FeedList = () => {
-  return <View />;
+  return (
+    <View style={styles.main}>
+      <FlatList
+        ItemSeparatorComponent={() => {
+          return <View style={styles.separator} />;
+        }}
+        showsVerticalScrollIndicator={false}
+        data={FeedData.data}
```

Comment:
We should order the feed starting with the most recent first.

Reply:
can we also handle "pull to refresh" functionality, loading state and also empty data

Reply:
we can implement this and the pull to refresh when we connect to live api